docs/api.rst

```diff
@@ -485,6 +485,55 @@ Possible Responses
 | 500 Server Error | Internal Raiden node error|
 +------------------+---------------------------+
 
+
+Querying token specific events
+------------------------------
+
+You can query for all new channels opened for a token by making a ``GET`` request to the following endpoint. ``GET /api/<version>/events/token/<token_address>``
```

Comment:
Hmm you are right that we are using plural already in the two endpoints you mention. I think we had this discussion in the past and indeed we had concluded with plural for the fact that they are collections.

Will address.